### DISCUSSION TITLE
[Bug]: error while loading EfficientAd model

### DISCUSSION THREAD

**Discussion by Rkoy145** [open-edge-platform/anomalib#1336] [Q&A]
### Describe the bug

The error message is "RuntimeError: Expected hasRecord("version") to be true, but got false."
It happens at line 
"self.model.teacher.load_state_dict(torch.load(teacher_path, map_location=torch.device(self.device)))"
in function prepare_pretrained_model
I've tried torch lightning from 1.7 to 1.10, and this error presists.

### Dataset

MVTec

### Model

Other (please specify in the field below)

### Steps to reproduce the behavior

Model:EfficientAd model
just run "python3 tools/train.py --model efficient_ad" and i get this error

### OS information

OS information:
Ubuntu 20.0
python 3.9.12
Anomalib version: 0.7.0
- PyTorch version: 1.12.1+cu116

### Expected behavior

Load model weight.

### Pip/GitHub

pip

### Configuration YAML

```yaml
The original YAML, with changed dataset path
```

### Logs

```shell
File "/opt/anaconda3/lib/python3.9/site-packages/anomalib/models/efficient_ad/lightning_model.py", line 105, in prepare_pretrained_model
self.model.teacher.load_state_dict(torch.load(teacher_path, map_location=torch.device(self.device)))
File "/opt/anaconda3/lib/python3.9/site-packages/torch/serialization.py", line 705, in loadwith _open_zipfile_reader(opened_file) as opened_zipfile:
File "/opt/anaconda3/lib/python3.9/site-packages/torch/serialization.py", line 242, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: Expected hasRecord("version") to be true, but got false.  (Could this error message be improved?  If so, please report an enhancement request to PyTorch.)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blaz-r** [comment]
Hello. I can't reproduce this. I recommend that you follow the getting started guide and get a fresh environment. If you are sure that your env is properly setup, something might be wrong with the weights. Try deleting the `pre_trained\efficientad_pretrained_weights` inside root folder so they are downloaded again.

**Comment by samet-akcay** [comment]
Hi @Rkoy145, moving this to Q&A as this does not seem to be an issue. Let's discuss this there.
